mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
If we can't be certain that the error was caused by inadequate oauth scope we should indicate that uncertainty in the error message. Instead of saying "you need to use one of the following oauth scopes", maybe we should say "make sure you are usinging one of the following oauth scopes" or phrase it as a question like "are you using at least one of the following oauth scopes?".